Copyfile stubs fail on alpine linux

Failure wasn't present in 3.20.

https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/a9b8a0d0db631d21208bd4ab7355e867d7ecf88d/variant/distributions,alpine-3.22-ocaml-4.14,dune.3.21.0~alpha2

```
# system               arch=x86_64 os=linux os-distribution=alpine os-version=3.22.2
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
-> retrieved dune.3.21.0~alpha2  (cached)
[ERROR] The compilation of dune.3.21.0~alpha2 failed at "ocaml boot/bootstrap.ml -j 255".

#=== ERROR while compiling dune.3.21.0~alpha2 =================================#
# context              2.5.0 | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/ocaml/dune/releases/download/3.21.0_alpha2/dune-3.21.0.alpha2.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/dune.3.21.0~alpha2
# command              ~/.opam/opam-init/hooks/sandbox.sh build ocaml boot/bootstrap.ml -j 255
# exit-code            2
# env-file             ~/.opam/log/dune-7-a73cd8.env
# output-file          ~/.opam/log/dune-7-a73cd8.out
### output ###
# ocamlc -output-complete-exe -intf-suffix .dummy -g -o .duneboot.exe -I boot unix.cma boot/types.ml boot/libs.ml boot/duneboot.ml
# ./.duneboot.exe -j 255
# cd _boot && /home/opam/.opam/4.14/bin/ocamlopt.opt -c -g -I +threads copyfile_stubs.c
# otherlibs/stdune/src/copyfile_stubs.c:67:10: fatal error: linux/version.h: No such file or directory
# compilation terminated.
```

--- a/otherlibs/stdune/src/copyfile_stubs.c
+++ b/otherlibs/stdune/src/copyfile_stubs.c
@@ -64,10 +64,10 @@ CAMLprim value stdune_sendfile(value v_in, value v_out, value v_size) {
 
 #include <sys/sendfile.h>
 #include <sys/utsname.h>
-#include <linux/version.h>
 #include <dlfcn.h>
 #include <stdio.h>
 
+#define KERNEL_VERSION(a, b, c) (((a) << 16) + ((b) << 8) + (c))
 #define FD_val(value) Int_val(value)
 
 CAMLprim value stdune_copyfile(value v_from, value v_to) {

